docs/source/python_spec.rst

```diff
@@ -134,6 +134,37 @@ C API which is called either when the refcount on the capsule named
 Note: the capsule names ``"dltensor"`` and ``"used_dltensor"`` must be
 statically allocated.
 
+The ``DLManagedTensor`` deleter must ensure that sharing beyond Python
+boundaries is possible, this means that the GIL must be acquired explicitly.
```

Comment:
aquired explicitly if the original memory comes from python runtime and deleter involves interacting with python runtime.

Reply:
Fair, added "[...] if it uses Python objects or API"